### PR TITLE
Remove unused Docker container code

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/zrq6ih4v/528-investigate-removal-of-no-processes-found-for-x-for-apps-with-a-healthcheck

Please see the commits for more details.

### CODE REVIEW

File: modules/govuk/manifests/node/s_cache.pp [hunk 1]
```diff
@@ -68,10 +68,6 @@
     content => "net.netfilter.nf_conntrack_tcp_timeout_established = 600\n",
   }
 
-  if $router_as_container {
-    include ::govuk_containers::apps::router
-  }
```
Comment (kevindew) [inline modules/govuk/manifests/node/s_cache.pp:73]: I think we also need to remove the `$router_as_container` variable

**Review by kevindew — COMMENTED** [review]
Looks good, nice clean-up.

File: modules/govuk/manifests/node/s_cache.pp (hunk 1 above)
Comment (benthorner) [reply modules/govuk/manifests/node/s_cache.pp:73]: Good spot - removed 👍

**Review by kevindew — APPROVED** [review]
Nice one

**Pull request merged**